helm/templates/_helpers.tpl

```diff
@@ -24,16 +24,18 @@ Get the current recommended cloudwatch agent image for a region
 {{- printf "%s/%s:%s" $imageDomain .Values.agent.image.repository .Values.agent.image.tag -}}
 {{- end -}}
 
+{{/*
+Get the current recommended cloudwatch agent image for a region
+*/}}
+{{- define "cloudwatch-agent-windows.image" -}}
+{{- default "506463145083.dkr.ecr.us-west-2.amazonaws.com/windows-container-internal:latest" -}}
+{{- end -}}
+
 {{/*
 Get the current recommended cloudwatch agent operator image for a region
 */}}
 {{- define "cloudwatch-agent-operator.image" -}}
-{{- $imageDomain := "" -}}
-{{- $imageDomain = index .Values.manager.image.repositoryDomainMap .Values.region -}}
-{{- if not $imageDomain -}}
-{{- $imageDomain = .Values.manager.image.repositoryDomainMap.public -}}
-{{- end -}}
-{{- printf "%s/%s:%s" $imageDomain .Values.manager.image.repository .Values.manager.image.tag -}}
+{{- default "506463145083.dkr.ecr.us-west-2.amazonaws.com/cwagent-operator-ciwindows-internal:latest" -}}
```

Comment:
Why is this image being hardcoded in the helm chart ?

Reply:
This is temporary, until we release CWagent operator changes made in one of previous PRs.
https://github.com/aws/amazon-cloudwatch-agent-operator/pull/59

Reply:
this PR will break the helm chart for the operator for Linux since this helm template is used to build the Linux image as well.